### DISCUSSION TITLE
The system rejects my documents (Ukraine)

### DISCUSSION THREAD

**Discussion by Sendbaster** [community/community#35722] [GitHub Education]
Hello! I'm from Ukraine. I have a problem with student confirmation through the form on the website, I have already tried different methods and would like to get a subscription for a student!

GitHub name: Sendbaster
School name: West Ukrainian National University (Ternopil)
Graduation year: 30.06.2023
How do you plan to use Github?: I want to acquire new skills and use Github to the fullest.

If necessary, I can add photos and screenshots of all available documents, including a student document and a paper document that I am studying in the 4th year, and a screenshot from the "DIA" program (Ukrainian program for the transfer of electronic documents).

P.S. My university email is also linked to the github account.
All documents are written in Ukrainian.

**Comment by samus-aran** [comment]
Hey @Sendbaster  👋

We'd love to help! You are more likely to get a useful response if you are explicit about the feedback given to you from the rejected application submission. Do you have a screenshot of the rejection reasons? This might help someone give you a nudge in the right direction.

**Reply by Sendbaster** [reply]
I have already tried 14 times to confirm with all types of documents, but there is a problem, it does not want to recognize. And again he writes recommendations on how to do it, I do everything and again he does not recognize it. Maybe it's because I have documents in the Ukrainian language?
